src/commands/build.ts

```diff
@@ -3,25 +3,21 @@ import { BundleService } from '../services/bundle-service'
 import { ComponentService } from '../services/component-service'
 import { BaseBuildCommand } from './base-build'
 import { Phase } from '../services/command-factory-service'
-import { ComponentType } from '../models/component'
+import { Component, ComponentType } from '../models/component'
 
 export default class Build extends BaseBuildCommand {
   static description = 'Build bundle components'
+  // Disable argument validation for variable length arguments (list of components)
```

Comment:
I read about this in the doc. Unfortunately this is causing the command help to be missing the arguments:
```
USAGE
  $ entando-bundle-cli run [--all-ms | --all-mfe | --all]

FLAGS
  --all      Run all the bundle components
  --all-mfe  Run all the bundle micro frontends
  --all-ms   Run all the bundle microservices

DESCRIPTION
  Run bundle components

EXAMPLES
  $ entando-bundle-cli run my-component

  $ entando-bundle-cli run my-component-1 my-component-2

  $ entando-bundle-cli run --all-ms

  $ entando-bundle-cli run --all-mfe

  $ entando-bundle-cli run --all
```
Can you check if there is a workaround to document the command correctly? 
It should be something like: 

`entando-bundle-cli run [component...] [--all-ms | --all-mfe | --all]`

ARGUMENTS
  components...  a list of components

cc @zonia3000

Reply:
You can define a static variable `usage`:
```
static usage = 'run [component...] [--all-ms | --all-mfe | --all]'
```

Reply:
done